internal/actions/runWorkflow.go

```diff
@@ -259,9 +254,13 @@ func finalize(inputs finalizeInputs) error {
 	case environment.ModeDirect:
 		var releaseInfo *releases.ReleasesInfo
 		if !inputs.SourcesOnly {
-			releaseInfo, err = getReleasesInfo()
-			if err != nil {
-				return err
+			releaseInfo = inputs.currentRelease
+			// TODO: Temporary until we figure out how to handle parsed terraform previous release version
```

Comment:
@ThomasRooney 
If you have any ideas here

Reply:
Just remove the TF publishing target -- AFAIK it's entirely unused. Everyone just uses goreleaser.